src/main/java/com/umc/networkingService/domain/member/entity/Member.java

```diff
@@ -105,8 +106,33 @@ public void updateContributionPoint(Long usedPoint) {
         else this.contributionPoint += usedPoint;
     }
 
+    public Semester getRecentSemester() {
+        List<SemesterPart> semesterParts = this.getSemesterParts();
+
+        List<Semester> semesters = semesterParts.stream()
+                .map(SemesterPart::getSemester)
+                .sorted(Comparator.comparingInt(Enum::ordinal))  // 순서대로 정렬
+                .collect(Collectors.toList());
+
+        return semesters.get(semesters.size() - 1);
```

Comment:
이 부분에서 .sorted(), .get() 대신 .max()를 사용하는 것이 더 효율적인 방법인 것 같아요!!!
```suggestion
        return Semester> semester = semesterParts.stream()
                .map(SemesterPart::getSemester)
                .max(Comparator.comparingInt(Enum::ordinal))
                .orElseThrow("예외처리"); // 값이 비어있을 경우에 대한 예외 처리
```